### ISSUE TITLE
Implement example mapping activities as APIs

### ISSUE THREAD

**Issue by turbomam** [cancerDHC/example-data#34]
The recent contract renewal (RFP?) mentions less new tool development and more API use cases

See also #31

**Comment by gaurav** [comment]
I disagree: while the RFP certainly wants us to do less tool development (at least partially because any tools we develop will need to be hosted on NCI infrastructure and need to continue to function beyond the lifetime of the project, see #35), they only mention APIs twice, in both cases relating to CRDC nodes making either "an API or a process that produces a data object that CDA can ingest" so that the CDA can ingest their data. Please do correct me if I'm wrong!

As of right now, I think we should be encouraging everybody to use a "data object" in the CRDCH Instance format (in YAML) to exchange data -- these are the types of objects we hand-created for the CCDH Pilot and are creating in the Example Data repo, and using this sort of instance data will allow people to use the LinkML-based validation and transformation tools to validate this data. Once the CDA has a better idea of how to incorporate our model into their ETL pipelines, I think we can revisit this question and see if an API for validation/transformation is actually necessary.

Now, we still have the CCDH Terminology Service to consider, and how we can eventually get that approved for an Authority to Operate OR come up with some way of rendering it unnecessary (e.g. by making the mapping data contained within it publicly accessible, maybe through SSSOM files). But I think we should discuss that in its repo :)
